patroni/dcs/__init__.py

```diff
@@ -381,13 +381,13 @@ def from_node(index, value):
                 data = {}
         else:
             data = {}
-        return SyncState(index, data.get('leader'), data.get('sync_standby'))
+        return SyncState(index, data.get('leader'), data.get('sync_standby', []))
```

Comment:
What about backward compatibility?
When replicas are upgraded they should still be able to read the data in the old format.

Reply:
We could possibly use a new key instead to store synchronous standbys list and leave the existing key to keep only one node info to provide backward compatibility for its peers during transition to new version. Please let me know if this is acceptable so I can fix the code accordingly. Thanks.